src/PlusDataCollection/USDigitalEncodersTracking/vtkPlusUSDigitalEncodersTracker.h

```diff
@@ -146,11 +146,11 @@ class vtkPlusDataCollectionExport vtkPlusUSDigitalEncodersTracker : public vtkPl
 
 
 private:  // Variables.
-    bool coreXY;
+    bool coreXY = false;

```

Comment:
The simple variables are initialized at place of declaration, the ones requiring function calls in the constructor. Initializing simple variables in the constructor requires more typing, makes maintenance slightly harder, and requires more effort to find out its default value (as it is separated from the declaration).